packages/@o3r/eslint-config/schematics/ng-add/vscode/index.ts

```diff
@@ -19,6 +19,12 @@ export const updateVscode: Rule = async () => {
       settings['eslint.useFlatConfig'] = true;
       settings['editor.defaultFormatter'] = 'dbaeumer.vscode-eslint';
       settings['editor.foldingImportsByDefault'] = true;
+      settings['eslint.rules.customizations'] = [
+        { rule: '@stylistic/*', fixable: true, severity: 'off' },
+        { rule: '!@stylistic/*', fixable: true, severity: 'info' }
+      ];
+      settings['editor.codeActionsOnSave'] ||= {};
+      (settings['editor.codeActionsOnSave'] as JsonObject)['source.fixAll'] = 'explicit';
```

Comment:
weird that you need an explicit cast only here, can't you instead force the type of `settings` on line 18 ?